app/storage/secureStorage.ts

```diff
@@ -193,13 +204,67 @@ export async function doDecryptWithPasscode(pass: string, salt: string, data: Bu
 export async function encryptAndStoreWithPasscode(address: string, pass: string, data: Buffer) {
     try {
         const passKey = await generateKeyFromPasscode(pass);
+
         storage.set(`${address}/${passcodeSaltKey}`, passKey.salt);
+
         const nonce = await getSecureRandomBytes(24);
         const sealed = sealBox(data, nonce, passKey.key);
         const encrypted = Buffer.concat([nonce, sealed]);
+
         storage.set(`${address}/${passcodeEncKey}`, encrypted.toString('base64'));
         storage.set(`${address}/${passcodeStateKey}`, PasscodeState.Set);
+
+        return encrypted;
     } catch (e) {
         throw Error('Unable to encrypt data with passcode');
     }
+}
+
+export function getBiometricsMigrated(isTestnet: boolean) {
+    return storage.getBoolean(`migrated-biometrics-enc-keys-${isTestnet ? 'testnet' : 'mainnet'}`);
+}
+
+export function migrateBiometricsEcnKeys(isTestnet: boolean) {
```

Comment:
migrateBiometricsEncKeys

Reply:
Great, thanks for the attention to typos!